code/modules/mob/living/carbon/xenomorph/abilities/burrower/burrower_powers.dm

```diff
@@ -96,6 +96,15 @@
 
 	if(!HAS_TRAIT(src, TRAIT_ABILITY_BURROWED))
 		to_chat(src, SPAN_NOTICE("We must be burrowed to do this."))
+
+	var/area/current_area = get_area(T)
+	if(current_area.flags_area & AREA_NOTUNNEL)
+		to_chat(src, SPAN_XENOWARNING("There's no way to surface here!!"))
+		return
+
+	if(!burrow)
```

Comment:
I didn't write this... am I going insane? 

I'm going to look at it tomorrow, but I'm quite certain this was already there.

Reply:
This is weird that it wrote like 105/106/107 in my pr... my local git doesn't say I wrote that so, that's bizzare.

Reply:
Lint is saying `burrow` is undefined, but it is defined in the `Xenomorph.dm` class and there is a macro in `Burrower.dm` in the `base_action` list. 

Is lint just unable to recognise these things correctly?